allure-generator/src/main/javascript/components/graph-pie-chart/PieChartView.js

```diff
@@ -33,7 +33,7 @@ class PieChartView extends BaseChartView {
       name: key.toUpperCase(),
       value: stats[key],
       part: stats[key] / total,
-    }));
+    })).filter(item => item.part !== 0);
```

Comment:
why do you filter by `part`, and not by `value`? Also it could be simplified to just `.filter(item => item.part)`